Extensions/ParticleSystem/particleemitterobject-pixi-renderer.js

```diff
@@ -0,0 +1,264 @@
+
```

Comment:
As in the Cocos2d-JS renderer, you can remove the extra blank lines, or add a comment/copyright comment :)